### PR TITLE
#343 Support displaying short agency name on itinerary

### PR DESCRIPTION
**Pull request by amenk**

### CODE REVIEW

File: lib/trufi_models.dart [hunk 1]
```diff
@@ -366,6 +366,7 @@ class PlanItineraryLeg {
     this.distance,
     this.duration,
     this.toName,
+    this.agencyName
```
Comment (RaimundWege) [inline lib/trufi_models.dart]:
```suggestion
    this.agencyName,
```
We leave an extra `,` at the end for automatic code formatting.

File: lib/trufi_models.dart [hunk 2]
```diff
@@ -421,7 +425,8 @@ class PlanItineraryLeg {
       _RouteLongName: routeLongName,
       _Distance: distance,
       _Duration: duration,
-      _To: {_Name: toName}
+      _To: {_Name: toName},
+      _AgencyName: agencyName
```
Comment (RaimundWege) [inline lib/trufi_models.dart]:
```suggestion
      _AgencyName: agencyName,
```

**Comment by amenk** [comment]
I might revert this as we changed our strategy for Addis Ababa and might use a single agency GTFS, so we might put the bus type in the route name

**Comment by mdoelker** [comment]
Does this still apply?

**Comment by amenk** [comment]
We don't use this anymore

**Pull request closed without merging**